docs/v0.5/getting-started/kind.md

```diff
@@ -0,0 +1,307 @@
+---
+id: kind
+title: Getting started on kind
+sidebar_label: kind
+---
+
+The following will help you get started running a riff function with Knative on [kind](https://kind.sigs.k8s.io/), a tool for running local Kubernetes clusters using Docker container “nodes”. 
+
+### Prerequisites
+These instructions assume that you are running recent versions of [go](https://golang.org/) and [docker](https://docs.docker.com/install/).
+You will also need a version of the kubectl CLI at least as recent as the kind node (v1.15.3).
+
+To install from released builds, see the [kind Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start).
+
+### Resize the VM and disable built-in Kubernetes
+
+If you are running Docker on macOS or Windows, use the Preferences feature in the Docker menu to open Advanced settings and configure your VM with at least 5GB of memory and 4 CPUs. Also, make sure that you don't have Kubernetes enabled. Click on Apply & Restart.
+
+### Install kind
+
```

Comment:
Can't we just link to https://kind.sigs.k8s.io/docs/user/quick-start#installation

Would prefer to not require that go is installed

Reply:
you probably don't need 5GB memory and 4 CPUs unless this project is really expensive, running kind works in the minimum spec, more resources are needed to build kubernetes for custom node images.

Reply:
linked to kind install page.